src/Model/Post.php

```diff
@@ -73,6 +73,14 @@ class Post extends Model {
 	 */
 	protected $post_type_object;
 
+	/**
+	 * The ID used to identify the fields' origin.
+	 *
+	 * @var array $fields
+	 * @access public
+	 */
+	protected $fields_id_name = 'post';
```

Comment:
@kidunot89 what is this for? I don't see it being used 🤔

Reply:
My bad. I forgot to remove it.